### PR TITLE
Jonmv/reindexing status in application v2

### PR DESCRIPTION
**Pull request by jonmv**
@bjorncs please review.

### CODE REVIEW

**Comment by jonmv** [comment]
PR build failed on something unrelated.

File: configserver/src/test/java/com/yahoo/vespa/config/server/http/v2/ApplicationHandlerTest.java [hunk 1]
```diff
@@ -408,21 +520,42 @@ private void assertApplicationExists(ApplicationId applicationId, Zone zone) thr
...
     }
 
+    private void reindexing(ApplicationId application, Method method, String expectedBody) throws IOException {
+        String reindexingUrl = toUrlPath(application, Zone.defaultZone(), true) + "/reindexing";
+        HttpResponse response = createApplicationHandler().handle(createTestRequest(reindexingUrl, method));
+        assertEquals(200, response.getStatus());
+        if (expectedBody != null) {
+            ByteArrayOutputStream out = new ByteArrayOutputStream();
+            response.render(out);
+            System.err.println(out);
```
Comment (bjorncs) [inline configserver/src/test/java/com/yahoo/vespa/config/server/http/v2/ApplicationHandlerTest.java:537]: Debug code accidentally included?

**Review by bjorncs — APPROVED** [review]

File: configserver/src/test/java/com/yahoo/vespa/config/server/http/v2/ApplicationHandlerTest.java (hunk 1 above)
Comment (jonmv) [reply configserver/src/test/java/com/yahoo/vespa/config/server/http/v2/ApplicationHandlerTest.java:537]: Oh yeah.

File: configserver/src/test/java/com/yahoo/vespa/config/server/http/v2/ApplicationHandlerTest.java [hunk 2]
```diff
@@ -408,21 +527,42 @@ private void assertApplicationExists(ApplicationId applicationId, Zone zone) thr
...
```
Comment (jonmv) [inline configserver/src/test/java/com/yahoo/vespa/config/server/http/v2/ApplicationHandlerTest.java:537]:
```suggestion
```

**Pull request merged**